e2e/smokeTest.spec.js

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test'
+
+const title = 'PreProd Pomootskey'
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:3000')
+})
+
+test('has main page', async ({ page }) => {
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle(title)
+})
+;[
```

Comment:
This formatting looks bad. It seems like this file is not checked by prettier.

Reply:
Reformated.